packages/runtime/container-runtime/src/test/mockContainerContext.ts

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AttachState, ICriticalContainerError } from "@fluidframework/container-definitions";
+import { IContainerContext } from "@fluidframework/container-definitions/internal";
+import {
+	ConfigTypes,
+	type IConfigProviderBase,
+	type ITelemetryBaseLogger,
+} from "@fluidframework/core-interfaces";
+import { ISummaryTree } from "@fluidframework/driver-definitions";
+import {
+	IDocumentStorageService,
+	ISummaryContext,
+	type ISnapshotTree,
+	MessageType,
+	type IVersion,
+} from "@fluidframework/driver-definitions/internal";
+import { MockLogger, mixinMonitoringContext } from "@fluidframework/telemetry-utils/internal";
+import {
+	MockAudience,
+	MockDeltaManager,
+	MockQuorumClients,
+} from "@fluidframework/test-runtime-utils/internal";
+
```

Comment:
This looks like it was probably moved from another file, maybe `packages/runtime/container-runtime/src/test/containerRuntime.spec.ts`? Can you delete that copy?

Reply:
This is left over from a previous iteration where I had added tests to containerRuntime.spec.ts. I removed this file.